### PR TITLE
Header and footer

### PR DESCRIPTION
**Pull request by YaronMiro**
#11 

- Adjust design of **header** and **footer**  desktop and mobile according to the standard.
- Display menu for **anonymous** user.
- Display menu for **logged** user.

Different menu anonymous  vs logged user
menu

------------
------------

**Desktop** - Header and footer menus
selection_040
selection_039

------------
------------

**Mobile** - Header and footer menus
selection_037
selection_036

### CODE REVIEW

**Comment by YaronMiro** [comment]
@amitaibu Ready for review.

File: src/elm/App/View.elm [hunk 1]
```diff
@@ -144,58 +137,59 @@ navbarLoggedIn address model =
...
       [ ("active", isActivePage model.activePage page) ]
 
-    childAddress =
-      Signal.forwardTo address App.Update.ChildUserAction
 
     hrefVoid =
       href "javascript:void(0);"
+
+
+    -- Menu links for "loggedin" user.
```
Comment (amitaibu) [inline src/elm/App/View.elm]: loggedin => auth

File: src/elm/App/View.elm [hunk 2]
```diff
@@ -144,58 +137,59 @@ navbarLoggedIn address model =
...
+    itemsLoogedIn =
```
Comment (amitaibu) [inline src/elm/App/View.elm]: typo :)

=> `itemsAuth`

File: src/elm/App/View.elm [hunk 3]
```diff
@@ -144,58 +137,59 @@ navbarLoggedIn address model =
...
+          ]
+      , li []
+          [ a [ hrefVoid, onClick address App.Update.Logout ]
+              [ text "Logout" ]
+          ]
+      ]
+
+
+    -- Menu links for "a"nonymous" user.
+    itemsAnonymous =
```
Comment (amitaibu) [inline src/elm/App/View.elm]: => `itemsAnon`

File: src/elm/App/View.elm [hunk 4]
```diff
@@ -144,58 +137,59 @@ navbarLoggedIn address model =
...
-    childAddress =
-      Signal.forwardTo address App.Update.ChildUserAction
 
     hrefVoid =
       href "javascript:void(0);"
+
+
+    -- Menu links for "loggedin" user.
+    itemsLoogedIn =
+      [ li []
```
Comment (amitaibu) [inline src/elm/App/View.elm]: this `li` looks like it could be a function

Comment (amitaibu) [inline src/elm/App/View.elm]: (but could be a follow up)

**Comment by YaronMiro** [comment]
Merged manually.

**Pull request closed without merging**